content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -516,26 +517,34 @@ work between Windows and Linux:
   node. They should be applied to all containers as a best practice if the operator
   wants to avoid overprovisioning entirely.
 * `securityContext.allowPrivilegeEscalation` -
-   not possible on Windows; none of the capabilities are hooked up
+   not possible on Windows; none of the capabilities are hooked up. Pod API
+   validation would fail when `IdentifyPodOS` featuregate is enabled.
```

Comment:
I think for now it might be less confusing if we left this section as is and added a new section just for describing the new behavior if `IdenfityPodOS`. 
Also these checks only happen if both the feature gate is enabled and the pod specifies `OS=windows` which i think should be called out as well. The current updates could cause some confusion if only the feature gate was enabled but the OS isn't specified.